Tools/machines/summit-olcf/summit_warpx.profile.example

```diff
@@ -12,12 +12,11 @@ module load cuda/11.3.1
 # optional: faster re-builds
 module load ccache
 
-# optional: for PSATD in RZ geometry support
-module load blaspp/2021.04.01-cpu
-module load lapackpp/2021.04.00-cpu
-
-# optional: for PSATD support (CPU only)
-#module load fftw/3.3.9
+# optional: for RZ+PSATD support
```

Comment:
I usually add this comment as on the left since it's less ambiguous (RZ || PSATD or RZ && PSATD)
```suggestion
# optional: for PSATD in RZ geometry support
```